### ISSUE TITLE
Support browser sync for the liked mods

### ISSUE THREAD

**Issue by tranzystorek-io** [orhun/CoolModFiles#51]
More of a discussion than a feature request as I'm not extremely well-versed in the topic.

I'm wondering if the browser couldn't be used for syncing liked mods between machines, thanks to things like Firefox Account. Browser plugin settings have a dedicated place in the API for e.g. syncing settings. If I understand correctly, CMF stores the mod list in local storage, maybe there's

a) a way to make that syncable
or
b) some other place a browser can use for that purpose?

**Comment by orhun** [comment]
Apparently there is a JS API for this: <https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync>

**Comment by tranzystorekk** [comment]
Yeah but it seems this requires an Add-on ID that is given to/obtained by extensions :/
